src/index.html

```diff
@@ -6,13 +7,67 @@
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <title>Product cards</title>
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
+  <!-- #endregion -->
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__content">
+        <div class="card__image"></div>
+
+        <div class="card__title">
+          <p class="card__title-text">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </p>
+        </div>
+
+        <div class="card__code">
+          <p class="card__code-text">Product code: 195434</p>
+        </div>
+
+        <div class="card__rating">
+          <div class="card__rating-stars card__rating-stars--1"></div>
+          <div class="card__rating-stars card__rating-stars--2"></div>
+          <div class="card__rating-stars card__rating-stars--3"></div>
+          <div class="card__rating-stars card__rating-stars--4"></div>
+          <div class="card__rating-stars card__rating-stars--5"></div>
```

Comment:
The way you've implemented stars for rating is not efficient. Consider using a loop in your SCSS to generate the stars, or use a single container with a background image that represents the stars, and control the filled portion with width and background clipping.